avalara_salestax/models/account_tax.py

```diff
@@ -52,7 +52,7 @@ def _get_compute_tax(
         # For check credential
         avalara_obj = AvaTaxService(
             config.account_number, config.license_key,
-            config.service_url, config.request_timeout, config.logging)
+            config.service_url.encode('ascii'), config.request_timeout, config.logging)
```

Comment:
Can you apply this everywhere the service url is passed to create a service? And mind the line length ;-)